external/DoCIF

```diff
@@ -1 +1 @@
-Subproject commit 59e96f6dbb269c8b9c4dfcf44bbdc60ff876927f
+Subproject commit e52def19124bec4c9b743a8f7276ef2b14f64b51
```

Comment:
@ashaw596 59e96f6dbb is the new one - it looks you merged master to get the new one, then added a commit reverting back to the old one.